Makefile

```diff
@@ -81,6 +81,10 @@ image: deps-development
 	-t $(REPOSITORY):$(BRANCH) \
 	-f $(APP_DIR)/Dockerfile \
 	.
+
+testing: image
```

Comment:
Was this committed by accident?

Reply:
No, I want it to be there and make developing easyer

Reply:
You missed the 2nd part of the comment :(